src/main.cpp

```diff
@@ -572,7 +572,7 @@ bool IsMay152018Next(const Consensus::Params &consensusparams, const CBlockIndex
     {
         return false;
     }
-
+printf("mining for value %ld\n", miningForkTime.value);
```

Comment:
this will probably make travis fail due to formatting checker (no big deal since is temporary)